added v12 to the files

oliviamyhre opened the pull request.

oliviamyhre committed:
added v12 to the files

taylomol000 committed:
Preliminary work

Separated days, fixed page numbers, added in ref="", fixed as many Catholic Latin words tagged as people/places as I could find (ie., Quintagesima, Lent), placed datelines and entries on separate lines, tagged "the girls" as a <personGrp> throughout the volume, added in <p> and </p> tags.

taylomol000 committed:
Added <div> tags; started on my section (Nov-Jan)

taylomol000 committed:
Couple tagging changes

hannah627 committed:
Update volume12.xml

Began work on my section, mainly adding datelines

hannah627 committed:
Update volume12.xml

Added datelines for all entries in my section

emmafritzberg commented on volume12.xml:
```diff
@@ -0,0 +1,856 @@
+          <persName ref="#Andrews_Emma_B">Emma B. Andrews </persName>
+        </author>
+        <editor>
+          <persName ref="SK">Sarah Ketchley </persName>
```
Should be `ref="#SK"`

emmafritzberg commented on volume12.xml:
```diff
@@ -0,0 +1,856 @@
+  </teiHeader>
+  <text>
+    <body>
+<!-- Molly Taylor -->
```
Will want to remove comments before merging into `main`

oliviamyhre committed:
update through may 19th

oliviamyhre committed:
Merge branch 'volume12' of https://github.com/eba-diary/tagged-volumes into volume12

hannah627 committed:
Update volume12.xml

Did some tagging and tag reorganization

hannah627 committed:
Update volume12.xml

Did more tagging work, especially around Arthur Weigall and various person groups

hannah627 committed:
Update volume12.xml

Did a lot more tagging

hannah627 committed:
Update volume12.xml

hannah627 committed:
Update volume12.xml

taylomol000 committed:
More work on my section

hannah627 committed:
Update volume12.xml

more tagging work

hannah627 committed:
Update volume12.xml

more tagging work

hannah627 committed:
Update volume12.xml

more tagging

oliviamyhre committed:
Update volume12.xml

oliviamyhre committed:
Merge branch 'volume12' of https://github.com/eba-diary/tagged-volumes into volume12

oliviamyhre committed:
Update volume12.xml

taylomol000 committed:
Merge branch 'volume12' of https://github.com/eba-diary/tagged-volumes into volume12

hannah627 committed:
Update volume12.xml

fixed mysterious deletion of all my work

hannah627 committed:
Update volume12.xml

changed a lot of personGrps to persNames; made a list of questions to answer to complete my section (9 long)

taylomol000 committed:
Update volume12.xml

taylomol000 committed:
Merge branch 'volume12' of https://github.com/eba-diary/tagged-volumes into volume12

hannah627 committed:
Update volume12.xml

got my work back and solved abbe and kons problems

taylomol000 committed:
Some minor edits

While going through and adding people to the Master Indices List, I've found a few errors here and there. Work in progress.

hannah627 committed:
Update volume12.xml

stashing changes to pull origin

hannah627 committed:
Merge branch 'volume12' of https://github.com/eba-diary/tagged-volumes into volume12

ketchley committed:
Update volume12.xml

hannah627 committed:
Update volume12.xml

solved menshia, birds, linton-smiths, and hooper girls issues

hannah627 committed:
Merge branch 'volume12' of https://github.com/eba-diary/tagged-volumes into volume12

hannah627 committed:
Update volume12.xml

solved the peckett/pickett/packett issue

oliviamyhre committed:
Update volume12.xml

oliviamyhre committed:
Merge branch 'volume12' of https://github.com/eba-diary/tagged-volumes into volume12

oliviamyhre committed:
Update volume12.xml

hannah627 committed:
Update volume12.xml

Made minor changes to fix errors found as I work through the XSLT

hannah627 committed:
Update volume12.xml

more minor changes as i work through the XSLT

hannah627 committed:
Update volume12.xml

made some corrections based on the new tagging guidelines

taylomol000 committed:
Committing stashed changes

taylomol000 committed:
Merge branch 'volume12' of https://github.com/eba-diary/tagged-volumes into volume12

oliviamyhre committed:
Update volume12.xml

ketchley committed:
Update volume12.xml

ketchley committed:
Update volume12.xml

ketchley committed:
Update volume12.xml

ketchley committed:
Update volume12.xml

ketchley committed:
Update volume12.xml

oliviamyhre committed:
Update volume12.xml

oliviamyhre committed:
updated dates

hannah627 committed:
Update volume12.xml

saving changes for the night - fixing lots of little errors caught by the XSLT

hannah627 committed:
Update volume12.xml

minor changes as I go through the XSLT and find things to fix

oliviamyhre committed:
pauline whiting to pauline whitney

emmafritzberg commented on volume12.xml:
```diff
@@ -0,0 +1,2071 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
@emmafritzberg to move volume12.xml to diary-volumes

emmafritzberg commented on volume14.xml:
```diff
@@ -0,0 +1,551 @@
+<TEI xmlns="http://www.tei-c.org/ns/1.0">
```
@emmafritzberg use lines 1-247 from this file for volume14

emmafritzberg committed:
Merge branch 'main' into volume12

emmafritzberg committed:
Merge branch 'main' into volume12

emmafritzberg committed:
move volume 12
